src/shared/approval-state/status-icon.js

```diff
@@ -0,0 +1,52 @@
+import { colors } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { getApprovalStateDisplayData } from './get-approval-state.js'
+import classes from './status-icon.module.css'
+
+const getIconColorForType = type => {
+    switch (type) {
+        case 'neutral':
+            return colors.blue800
+        case 'positive':
+            return colors.green800
+        case 'negative':
+            return colors.red800
+    }
+}
+
+const StatusIcon = ({ approvalState, showTitle }) => {
+    const {
+        icon: Icon,
+        displayName,
+        type,
+    } = getApprovalStateDisplayData(approvalState)
+
+    return (
+        <span
+            title={showTitle ? displayName : ''}
```

Comment:
Do we want to use the `title` attribute or a `@dhis2/ui` tooltip?

Reply:
I think Joe meant a `@dhis2/ui` tooltip, but I don't think it's a big deal. It'd be best to wrap it in a tooltip but if you are worried about performance or anything, it's not strictly required.